src/main/resources/org/jenkinsci/plugins/categorizedview/CategorizedJobsView/configure-entries.jelly

```diff
@@ -84,10 +84,10 @@ THE SOFTWARE.
  <script>
    (function() {
      Behaviour.specify("#recurse", 'ListView', 0, function(e) {
-       var nestedElements = $$('SPAN.nested')
+       var nestedElements = document.querySelectorAll('SPAN.nested')
        e.onclick = function() {
-         nestedElements.each(function(el) {
-           e.checked ? el.show() : el.hide();
+         nestedElements.forEach(function(el) {
+           el.style.display = e.checked ? "" : "none";

```

Comment:
Copypasta of `core/src/main/resources/hudson/model/ListView/configure-entries-resources.js` from Jenkins core.